x-pack/plugins/security_solution/server/endpoint/routes/actions/list.ts

```diff
@@ -38,4 +41,17 @@ export function registerActionListRoutes(
       actionListHandler(endpointContext)
     )
   );
+  // TODO: This route is a temporary solution until we decide on how RBAC should look like for Actions in Alerts
+  router.get(
+    {
+      path: BASE_ENDPOINT_ACTION_ALERTS_ROUTE,
```

Comment:
Ok. this just cleared up my prior command on the `const`. it should be redefined to be a `*/action/alerts`  route

Reply:
I'll enable it only when the flag is provided 👍 

We can't use the current one, because we do not meet the criteria when eg. we have no license. 

{ any: ['canReadActionsLogManagement', 'canAccessEndpointActionsLogManagement'] },

There is a ticket to discuss the RBAC for this route, I'll add it to the office hours.